bob-backend/src/core.rs

```diff
@@ -335,15 +337,23 @@ impl Backend {
     }
 
     pub async fn get(&self, key: BobKey, options: &BobOptions) -> Result<BobData, Error> {
-        let (vdisk_id, disk_path) = self.mapper.get_operation(key);
+        let (vdisk_id, disk_paths) = self.mapper.get_operation(key);
 
         // we cannot get data from alien if it belong this node
 
         if options.get_normal() {
-            if let Some(path) = disk_path {
+            if let Some(paths) = disk_paths {
                 trace!("GET[{}] try read normal", key);
-                self.get_local(key, Operation::new_local(vdisk_id, path.clone()))
-                    .await
+                let mut futures: FuturesUnordered<_> = paths.into_iter().map(|path| {
```

Comment:
Most of the time we will have one element in the `paths` collection. So using `FuturesUnordered<_>` will slow down 99% of practical use cases. I suggest using a normal for loop to simplify the code and avoid performance degradation

Reply:
As I wrote, `for` loop would be better here:
```
for path in paths {
    self.get_local(...).await
}
```